bukkit/src/main/java/net/william278/husksync/BukkitHuskSync.java

```diff
@@ -124,7 +124,7 @@ public void onEnable() {
 
             // Prepare database connection
             this.database = new MySqlDatabase(this);
-            log(Level.INFO, "Attempting to establish connection to the database...");
+            log(Level.INFO, "Attempting to establish connection to the database type " + settings.getSqlType().getDisplayName() + "...");
```

Comment:
```suggestion
            log(Level.INFO, "Attempting to establish connection to the " + settings.getSqlType().getDisplayName() + " database...");
```
nit: This wording is slightly clunky -- this may be a bit better.